### PR TITLE
About organisations stipends

### PR DESCRIPTION
**Pull request by rgaiacs**
I talked with @aterrel about this in December.

@aterrel Can you review the wording?

@kain88-de Can you let mentors know about this after we merge it?

### CODE REVIEW

File: organization/stipend.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Organization Stipend
+
+At the end of the Program,
+mentoring organizations,
+**in this case NumFOCUS**,
+may receive a $500 USD stipend per student mentored.
+
+NumFOCUS agrees to pass along stipend per student mentored
+to the project who mentored the student.
```
Comment (kain88-de) [inline organization/stipend.md]: This is a very generous rule. Thanks.

**Comment by kain88-de** [comment]
ping @aterrel

File: organization/stipend.md (hunk 1 above)
Comment (orbeckst) [inline organization/stipend.md]: This *is* generous. Thank you!

**Comment by kain88-de** [comment]
ping @aterrel

**Pull request merged**